src/jit/codegenarm64.cpp

```diff
@@ -3698,7 +3698,7 @@ CodeGen::genLclHeap(GenTreePtr tree)
         regMaskTP pspSymRegMask = genFindLowestBit(tmpRegsMask);
         tmpRegsMask &= ~pspSymRegMask;
         pspSymReg = genRegNumFromMask(pspSymRegMask);
-        getEmitter()->emitIns_R_S(ins_Store(TYP_I_IMPL), EA_PTRSIZE, pspSymReg, compiler->lvaPSPSym, 0);
```

Comment:
Similar correction is needed in codegenxarch.cpp?  It works because ins_Store(TYP_I_IMPL) == ins_Load(TYP_I_IMPL) on xarch.